### ISSUE TITLE
Add VisibleBoundsPadding to default template

### ISSUE THREAD

**Issue by davidjohnoliver** [unoplatform/uno#5246]
## What would you like to be added:
Set [VisibleBoundsPadding](https://platform.uno/docs/articles/features/VisibleBoundsPadding.html) in the default Uno template

## Why is this needed:
Apps don't display properly on iPhones and notched Android devices without it

## For which Platform:

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

**Comment by xperiandri** [comment]
If you approve we can handle that

**Comment by davidjohnoliver** [comment]
@xperiandri go for it!

**Comment by MartinZikmund** [comment]
@kazo0 I think this can now be closed, as this is part of the new Uno.Templates, correct?

**Comment by kazo0** [comment]
@MartinZikmund I don't believe we use VisibleBoundsPadding or SafeArea as part of the new templates

<img width="287" alt="image" src="https://github.com/unoplatform/uno/assets/4793020/38730d3f-e5c4-4bb5-8c69-d8d969a46ff5">

**Comment by MartinZikmund** [comment]
Ah! We probably should then 🤔
